src/App.js

```diff
@@ -118,6 +116,8 @@ const App = () => {
                 onSuccess={onSuccess}
                 onError={onError}
                 secure={secureFlag}
+                clientSdkId={""}
```

Comment:
Do you think it is worth using a webpack plugin to get the value from `process.env.SDK_ID` for the `clientSdkId`?

Reply:
Sure, already got the value from the `.env.local`.